src/update_handler.c

```diff
@@ -845,6 +846,13 @@ static gboolean copy_block_hash_index_image_to_dev(RaucImage *image, RaucSlot *s
 			target_written->invalid_from = c+1;
 			target_old->invalid_below = c;
 		}
+
+		percent = c * 100 / chunk_count;
```

Comment:
@BubuOT Thank you for addressing this!

Since `c` counts up to `chunk_count` - 1 in the for loop, I guess we have to use

```suggestion
		percent = (c + 1) * 100 / chunk_count;
```

to reach 100% for the last chunk.